packages/core/src/App/Containers/Modals/trading-experience-modal.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { connect } from 'Stores/connect';
+import { Button, Icon, Modal, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+
+const TradingExperienceModal = ({
+    cfd_score,
+    is_trading_experience_incomplete,
+    setShouldShowTradingAssessmentModal,
+    should_show_trading_assessment_modal,
+    setShouldShowTradeAssessmentForm,
+}) => {
+    React.useEffect(() => {
+        setShouldShowTradingAssessmentModal(cfd_score === 0);
+    }, []);
+
+    const handleOnSubmit = () => {
+        setShouldShowTradingAssessmentModal(false);
+        setShouldShowTradeAssessmentForm(true);
+    };
+
+    return (
+        <Modal
+            width='44rem'
+            className='center-risk-modal'
+            is_open={is_trading_experience_incomplete && should_show_trading_assessment_modal}
+        >
+            <Modal.Body>
+                <Icon icon='IcCurrencyEurCheck' size={95} />
+                <Text as='p' size='s' align='center' weight='bold' className='verified-account__text'>
```

Comment:
`s` is the default value
```suggestion
                <Text as='p' align='center' weight='bold' className='verified-account__text'>
```